images/installer-origin-release/Dockerfile.ci

```diff
@@ -0,0 +1,12 @@
+# This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
+
+FROM  quay.io/coreos/origin-release:bazel
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+
+ENV USER="bazel"
+ENV HOME="/tmp"
+
+RUN ls -al && \
+    /usr/bin/bazel --output_base=/tmp build smoke_tests && \
+    cp bazel-bin/tests/smoke/linux_amd64_stripped/go_default_test /tmp/smoke
```

Comment:
To avoid cluttering a volume-mounted host directory with random cruft, I'd rather create `/output` or some such, and volume-mount something there to receive this output executable.